azureDeploy.json

```diff
@@ -370,7 +370,7 @@
1QgTlVMTCwNCiAgYWN0aXZhdGlvbiB2YXJjaGFyKDIwKSBOT1QgTlVMTCwNCiAgYW10X3Bhc3N3b3JkIHZhcmNoYXIoNDApLA0KICBnZW5lcmF0ZV9yYW5kb21fcGFzc3dvcmQgQk9PTEVBTiBOT1QgTlVMTCwNCiAgY2lyYV9jb25maWdfbmFtZSBjaXRleHQsDQogIEZPUkVJR04gS0VZIChjaXJhX2NvbmZpZ19uYW1lLHRlbmFudF9pZCkgIFJFRkVSRU5DRVMgY2lyYWNvbmZpZ3MoY2lyYV9jb25maWdfbmFtZSx0ZW5hbnRfaWQpLA0KICBjcmVhdGlvbl9kYXRlIHRpbWVzdGFtcCwNCiAgY3JlYXRlZF9ieSB2YXJjaGFyKDQwKSwNCiAgbWVieF9wYXNzd29yZCB2YXJjaGFyKDQwKSwNCiAgZ2VuZXJhdGVfcmFuZG9tX21lYnhfcGFzc3dvcmQgQk9PTEVBTiBOT1QgTlVMTCwNCiAgdGFncyB0ZXh0W10sDQogIGRoY3BfZW5hYmxlZCBCT09MRUFOLA0KICB0ZW5hbnRfaWQgdmFyY2hhcigzNikgTk9UIE5VTEwsDQogIHRsc19tb2RlIGludGVnZXIgTlVMTCwNCiAgdXNlcl9jb25zZW50IHZhcmNoYXIoNykgTlVMTCwNCiAgaWRlcl9lbmFibGVkIEJPT0xFQU4gTlVMTCwNCiAga3ZtX2VuYWJsZWQgQk9PTEVBTiBOVUxMLA0KICBzb2xfZW5hYmxlZCBCT09MRUFOIE5VTEwsDQogIHRsc19zaWduaW5nX2F1dGhvcml0eSB2YXJjaGFyKDQwKSBOVUxMLA0KICBpZWVlODAyMXhfcHJvZmlsZV9uYW1lIGNpdGV4dCwNCiAgRk9SRUlHTiBLRVkgKGllZWU4MDIxeF9wcm9maWxlX25hbWUsdGVuYW50X2lkKSAgUkVGRVJFTkNFUyBpZWVlODAyMXhjb25maWdzKHByb2ZpbGVfbmFtZSx0ZW5hbnRfaWQpLA0KICBQUklNQVJZIEtFWSAocHJvZmlsZV9uYW1lLCB0ZW5hbnRfaWQpDQopOw0KQ1JFQVRFIFRBQkxFIElGIE5PVCBFWElTVFMgcHJvZmlsZXNfd2lyZWxlc3Njb25maWdzKA0KICB3aXJlbGVzc19wcm9maWxlX25hbWUgY2l0ZXh0LA0KICBwcm9maWxlX25hbWUgY2l0ZXh0LA0KICBGT1JFSUdOIEtFWSAod2lyZWxlc3NfcHJvZmlsZV9uYW1lLHRlbmFudF9pZCkgIFJFRkVSRU5DRVMgd2lyZWxlc3Njb25maWdzKHdpcmVsZXNzX3Byb2ZpbGVfbmFtZSx0ZW5hbnRfaWQpLA0KICBGT1JFSUdOIEtFWSAocHJvZmlsZV9uYW1lLHRlbmFudF9pZCkgIFJFRkVSRU5DRVMgcHJvZmlsZXMocHJvZmlsZV9uYW1lLHRlbmFudF9pZCksDQogIHByaW9yaXR5IGludGVnZXIsDQogIGNyZWF0aW9uX2RhdGUgdGltZXN0YW1wLA0KICBjcmVhdGVkX2J5IHZhcmNoYXIoNDApLA0KICB0ZW5hbnRfaWQgdmFyY2hhcigzNikgTk9UIE5VTEwsDQogIFBSSU1BUlkgS0VZICh3aXJlbGVzc19wcm9maWxlX25hbWUsIHByb2ZpbGVfbmFtZSwgcHJpb3JpdHksIHRlbmFudF9pZCkNCik7DQpDUkVBVEUgVEFCTEUgSUYgTk9UIEVYSVNUUyBkb21haW5zKA0KICBuYW1lIGNpdGV4dCBOT1QgTlVMTCwNCiAgZG9tYWluX3N1ZmZpeCBjaXRleHQgTk9UIE5VTEwsDQogIHByb3Zpc2lvbmluZ19jZXJ0IHRleHQsDQogIHByb3Zpc2lvbmluZ19jZXJ0X3N0b3JhZ2VfZm9ybWF0IHZhcmNoYXIoNDApLA0KICBwcm92aXNpb25pbmdfY2VydF9rZXkgdGV4dCwNCiAgY3JlYXRpb25fZGF0ZSB0aW1lc3RhbXAsDQogIGNyZWF0ZWRfYnkgdmFyY2hhcig0MCksDQogIHRlbmFudF9pZCB2YXJjaGFyKDM2KSBOT1QgTlVMTCwNCiAgQ09OU1RSQUlOVCBkb21haW5zdWZmaXggVU5JUVVFKGRvbWFpbl9zdWZmaXgsdGVuYW50X2lkKSwNCiAgUFJJTUFSWSBLRVkgKG5hbWUsIHRlbmFudF9pZCkNCik7DQpDUkVBVEUgVU5JUVVFIElOREVYIGxvd2VyX25hbWVfc3VmZml4X2lkeCBPTiBkb21haW5zICgobG93ZXIobmFtZSkpLCAobG93ZXIoZG9tYWluX3N1ZmZpeCkpKTsNCg0KQ1JFQVRFIERBVEFCQVNFIG1wc2RiOw0KDQpcY29ubmVjdCBtcHNkYg0KDQpDUkVBVEUgVEFCTEUgSUYgTk9UIEVYSVNUUyBkZXZpY2VzKA0KICAgICAgZ3VpZCB1dWlkIE5PVCBOVUxMLA0KICAgICAgdGFncyB0ZXh0W10sDQogICAgICBob3N0bmFtZSB2YXJjaGFyKDI1NiksDQogICAgICBtcHNpbnN0YW5jZSB0ZXh0LCANCiAgICAgIGNvbm5lY3Rpb25zdGF0dXMgYm9vbGVhbiwNCiAgICAgIG1wc3VzZXJuYW1lIHRleHQsDQogICAgICB0ZW5hbnRpZCB2YXJjaGFyKDM2KSwgTk9UIE5VTEwsDQogICAgICBmcmllbmRseW5hbWUgdmFyY2hhcigyNTYpLA0KICAgICAgZG5zc3VmZml4IHZhcmNoYXIoMjU2KSwNCiAgICAgIENPTlNUUkFJTlQgZGV2aWNlX2d1aWQgVU5JUVVFKGd1aWQpLA0KICAgICAgUFJJTUFSWSBLRVkgKGd1aWQsIHRlbmFudGlkKQ0KICAgICk7IA=="
```

Comment:
When I decode the code , it is showing as CREATE TABLE IF NOT EXISTS devices(
      guid uuid NOT NULL,
      tags text[],
      hostname varchar(256),
      mpsinstance text, 
      connectionstatus boolean,
      mpsusername text,
      tenantid varchar(36), NOT NULL,
      friendlyname varchar(256),
      dnssuffix varchar(256),
      CONSTRAINT device_guid UNIQUE(guid),
      PRIMARY KEY (guid, tenantid)
    );